src/Libraries/Microsoft.Extensions.ServiceDiscovery.Yarp/README.md

```diff
@@ -0,0 +1,42 @@
+# Microsoft.Extensions.ServiceDiscovery.Yarp
```

Comment:
We're adding YARP-specific libraries to dotnet/extensions? That wouldn't be better in the YARP repo?

Reply:
@eerhardt and I chatted a bit about this offline. I think both extensions and yarp are on the "same layer" of the stack, and there is value in keeping all of the ServiceDiscovery source together (similar to how we think there is value in keeping all of the MEAI code together). It will also allow things to version together, and for aspire to consume from a single place. I suggest we keep this here in extensions for now, and we can re-evaluate in the future if we want to move some of these elsewhere.